File: Math/Medium/2221/2221.py
from typing import List

class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        while len(nums) != 1:
            numLen = len(nums) - 1
            for i in range(numLen):
                nums[i] = (nums[i] + nums[i+1]) % 10
        return nums[0]

File: Math/Medium/2221/2221.c
int triangularSum(int* nums, int numsSize){
    while(numsSize != 1){
        numsSize--;
        for(int i = 0; i < numsSize; i++){
            nums[i] = (nums[i] + nums[i+1]) % 10;
        }
    }
    return nums[0];
}